NFIQ2/NFIQ2Algorithm/src/features/FJFXMinutiaeQualityFeatures.cpp

```diff
@@ -330,10 +199,10 @@ FJFXMinutiaeQualityFeature::computeMuMinQuality(
 	// iterate through all minutiae positions and
 	// compute own minutiae quality values
 	// based on block-wise Mu computation around FJFX minutiae location
-	for (unsigned int i = 0; i < minCount; i++) {
+	for (unsigned int i = 0; i < minutiaCount; i++) {
 		MinutiaData minData;
-		int x = (int)fmds[i]->x_coord;
-		int y = (int)fmds[i]->y_coord;
+		int x = (int)minutiaData.get()[i].x;
+		int y = (int)minutiaData.get()[i].y;
 		minData.x = x;
 		minData.y = y;
```

Comment:
Doesn't this work + do we need the temporary variables anymore?

```suggestion
		minData.x = static_cast<int>(minutiaData[i].x);
		minData.y = static_cast<int>(minutiaData[i].y);
```

Reply:
We can take it out, just need to update the use of x and y to minData.x and minData.y